### PR TITLE
switch to use vertica-ce docker image for CI test

### PR DESCRIPTION
**Pull request by blackeyepanda**

### CODE REVIEW

**Review by DMickens — APPROVED** [review]
This is good. Now all 3 open source drivers are up to date.

**Comment by sitingren** [comment]
@blackeyepanda I found there are nondeterministic failures in both go client and python client after switching to vertica-ce docker image. Such as the TLS configuration may not take effect in the next go client connection, or a query cancel from python client may not take effect in the server. I suspect the docker container setting or docker image itself have some problems.

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -18,16 +18,20 @@ jobs:
...
-          docker build -f docker-vertica/Dockerfile.centos.7_9.x --build-arg VERTICA_PACKAGE=vertica-ce.latest.rpm -t jbfavre/vertica docker-vertica
-          docker images
-          docker run --hostname vertica-docker --name vertica_docker -d -p 5433:5433 jbfavre/vertica
-          sleep 60
+          docker pull vertica/vertica-ce:12.0.0-0
+          docker run -d -p 5433:5433 -p 5444:5444 \
+            --name vertica_docker \
+            vertica/vertica-ce
+          echo "Vertica startup ..."
+          until docker exec vertica_docker /opt/vertica/bin/vsql -c "SELECT 1" > /dev/null 2>&1; do \
```
Comment (sitingren) [inline .github/workflows/ci.yaml:29]: The nondeterministic failures comes from this check. During the docker container startup, it first creates a database (`SELECT 1` works at this stage), then loads VMart data and start agent. The whole process takes about 90 secs. This causes the golang tests start too early and conflict with container startup.

Comment (blackeyepanda) [reply .github/workflows/ci.yaml:29]: nice catch, thanks for figuring this out.

**Pull request closed without merging**